spring-pulsar-spring-boot-autoconfigure/src/main/java/org/springframework/pulsar/autoconfigure/PulsarProperties.java

```diff
@@ -131,25 +132,23 @@ public static class Consumer {
 		/**
 		 * Number of messages that can be accumulated before the consumer calls "receive".
 		 */
-		private int receiverQueueSize = 1000;
+		private Integer receiverQueueSize = 1000;
 
 		/**
-		 * Time to group acknowledgements before sending them to the broker in
-		 * microseconds.
+		 * Time to group acknowledgements before sending them to the broker.
 		 */
-		private long acknowledgementsGroupTimeMicros = TimeUnit.MILLISECONDS.toMicros(100);
+		private Duration acknowledgementsGroupTime = Duration.ofMillis(100);
 
 		/**
-		 * Delay before re-delivering messages that have failed to be processed in
-		 * microseconds.
+		 * Delay before re-delivering messages that have failed to be processed.
 		 */
-		private long negativeAckRedeliveryDelayMicros = TimeUnit.MINUTES.toMicros(1);
```

Comment:
There is a reference to this property in `spring-pulsar-docs/src/main/asciidoc/pulsar.adoc` (lines 941,951). Can you take a scan through that for any property name changes in here? We have too many hardcoded code samples in the doc that we intend to migrate out to source code snippets at some point - in the meantime we just have to search manually when we rename things.

Reply:
done